crates/sui-bridge/src/sui_transaction_builder.rs

```diff
@@ -0,0 +1,225 @@
+// Copyright (c) Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+use std::{collections::HashMap, str::FromStr};
+
+use fastcrypto::traits::ToFromBytes;
+use move_core_types::ident_str;
+use once_cell::sync::OnceCell;
+use sui_types::{
+    base_types::{ObjectID, ObjectRef, SequenceNumber, SuiAddress},
+    programmable_transaction_builder::ProgrammableTransactionBuilder,
+    transaction::{ObjectArg, TransactionData},
+    TypeTag,
+};
+
+use crate::{
+    error::{BridgeError, BridgeResult},
+    types::{BridgeAction, TokenId, VerifiedCertifiedBridgeAction},
+};
+
+// TODO: once we have bridge package on sui framework, we can hardcode the actual
+// bridge dynamic field object id (not 0x9 or dynamic field wrapper) and update
```

Comment:
Is this function to get the bridge object or the bridge package?

Reply:
good catch. this is for bridge package